.travis-script.sh

```diff
@@ -1,48 +1,57 @@
-sudo apt-get --yes update
+# All apt-get installs
+sudo apt-get --yes update # Update
```

Comment:
Still an unnecessary comment here

Reply:
That comment is just to distinguish the installs from the update. I can remove that as well if you insist

Reply:
Yes thanks